src/igvfd/schemas/mixins.json

```diff
@@ -290,5 +290,14 @@
             "type": "string",
             "pattern": "^(\\S+(\\s|\\S)*\\S+|\\S)$"
         }
+    },
+    "additional_description": {
+        "additional_description": {
```

Comment:
I like the name additional_description, and we can leave it, I just wanted to point out that keeping the names of the previous properties enables some of the existing UI code to more easily be transferred.  That is not a big enough reason to swap this back to "description".